wgengine/router/runner.go

```diff
@@ -23,7 +24,9 @@ type commandRunner interface {
 	output(...string) ([]byte, error)
 }
 
-type osCommandRunner struct{}
+type osCommandRunner struct {
+	ambientCaps []uintptr
```

Comment:
This is now a value type with pointers in it, which isn't my favorite (https://github.com/golang/go/wiki/CodeReviewComments#receiver-type)

Maybe just a `ambientCapNetAdmin bool` instead of a slice?

And add a comment regardless.